apiserver/apiserver/coordinator/compilation.py

```diff
@@ -154,7 +160,7 @@ def update_compilation_status():
                     language=language,
                     version_number=model.bots.c.version_number + 1,
                     games_played=0,
-                    sigma=8.333,
+                    sigma=new_sigma,
```

Comment:
Still working through this, but I'm thinking games_played shouldn't be reset either. Not positive on that, but I think it makes sense since we have challenge games avoid incrementing that count now.

Reply:
Hmm, we are incrementing the version number though, and it is a different bot. Maybe it'd make more sense to have challenge games increment the count in this mode.

Reply:
I guess it depends somewhat on, is the number of games you played in the finals going to be a point of pride or 'badge' of any sort for people going forward? I notice that it is one of the three things we display on current profiles as a stat from last year. But then of course so is number of submissions, i.e. version number. *shrug*